hub/tests/fixtures/geocoding_cases.py

```diff
@@ -0,0 +1,295 @@
+geocoding_cases = [
+    # Name matching; cases that historically didn't work
+    {
+        "id": "1",
+        "council": "Barnsley",
+        "ward": "St Helens",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05000993",
+    },
+    {
+        "id": "2",
+        "council": "North Lincolnshire",
+        "ward": "Brigg & Wolds",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05015081",
+    },
+    {
+        "id": "3",
+        "council": "Test Valley",
+        "ward": "Andover Downlands",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05012085",
+    },
+    {
+        "id": "4",
+        "council": "North Warwickshire",
+        "ward": "Baddesley and Grendon",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05007461",
+    },
+    # Name rewriting required
+    {
+        "id": "5",
+        "council": "Herefordshire, County of",
+        "ward": "Credenhill",
+        "expected_area_type_code": "WD23",
+        "expected_area_gss": "E05012957",
+    },
```

Comment:
suggestion: have a "X city council" vs "X council" case here

Reply:
Not sure I follow, can you expand?